airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/manifest.yaml

```diff
@@ -1167,6 +1183,1001 @@ definitions:
+                  "unitsSoldOtherSku1d",
+                  "unitsSoldOtherSku7d",
+                  "unitsSoldOtherSku14d",
+                  "unitsSoldOtherSku30d",
+                  "salesOtherSku1d",
+                  "salesOtherSku7d",
+                  "salesOtherSku14d",
+                  "salesOtherSku30d",
+                ]
+              reportTypeId: spPurchasedProduct
+              filters: []
+              timeUnit: SUMMARY
+              format: GZIP_JSON
+    sponsored_products_asins_targets_report_stream:
+      type: DeclarativeStream
+      name: sponsored_products_asins_targets_report_stream
+      primary_key: ["profileId", "reportDate", "adGroupName", "advertisedAsin"]
+      incremental_sync:
+        $ref: "#/definitions/incremental_sync_report_datetime_cursor"
+      transformations:
+        $ref: "#/definitions/transformation_report_add_fields"
+      retriever:
+        $ref: "#/definitions/basic_async_retriever"
+        creation_requester:
+          $ref: "#/definitions/basic_async_retriever/creation_requester"
+          request_body_json:
+            name: "asins_targets report {{ stream_slice.start_time }}"
+            startDate: "{{ stream_slice.start_time }}"
+            endDate: "{{ stream_slice.end_time }}"
+            configuration:
+              adProduct: SPONSORED_PRODUCTS
+              groupBy: ["asin"]
+              columns:
+                [
+                  "campaignName",
+                  "campaignId",
+                  "adGroupName",
+                  "adGroupId",
+                  "advertisedAsin",
+                  "purchasedAsin",
+                  "advertisedSku",
+                  "campaignBudgetCurrencyCode",
+                  "matchType",
+                  "unitsSoldClicks1d",
+                  "unitsSoldClicks7d",
+                  "unitsSoldClicks14d",
+                  "unitsSoldClicks30d",
+                  "unitsSoldOtherSku1d",
+                  "unitsSoldOtherSku7d",
+                  "unitsSoldOtherSku14d",
+                  "unitsSoldOtherSku30d",
+                  "salesOtherSku1d",
+                  "salesOtherSku7d",
+                  "salesOtherSku14d",
+                  "salesOtherSku30d",
+                  "keywordId",
+                  "targeting",
+                  "keywordType",
+                ]
+              reportTypeId: spPurchasedProduct
+              filters: []
+              timeUnit: SUMMARY
+              format: GZIP_JSON
+
+  incremental_sync_report_datetime_cursor:
+    type: DatetimeBasedCursor
+    cursor_field: reportDate
+    cursor_datetime_formats:
+      - "%Y-%m-%d"
+    datetime_format: "%Y-%m-%d"
+    lookback_window: P{{ config.get('look_back_window', 3) }}D
+    start_datetime:
+      type: MinMaxDatetime
+      datetime: "{{ config.get('start_date', today_utc().strftime('%Y-%m-%d') ) }}"
+      datetime_format: "%Y-%m-%d"
+    end_datetime:
+      datetime: "{{ today_utc().strftime('%Y-%m-%d') }}"
```

Comment:
nit, do we need to convert `today_utc()` into a YMD string? It should work w/o it right?

Reply:
you're right, it works!
refactored